pmd-core/src/main/java/net/sourceforge/pmd/AbstractPropertySource.java

```diff
@@ -127,25 +118,22 @@ public boolean hasDescriptor(PropertyDescriptor<?> descriptor) {
         return result;
     }
 
-    /**
-     * {@inheritDoc}
-     */
+
     @Override
     public <T> void setProperty(PropertyDescriptor<T> propertyDescriptor, T value) {
```

Comment:
Hum we actually can't implement the first version since `setProperty(PropertyDescriptor<T>, T)` has the same erasure as `setProperty(PropertyDescriptor<List<T>>, T)`. We'll go for the second version